llama_index/storage/storage_context.py

```diff
@@ -67,15 +80,19 @@ def from_defaults(
             vector_store = vector_store or SimpleVectorStore.from_persist_dir(
                 persist_dir, fs=fs
             )
+            graph_store = graph_store or SimpleGraphStore.from_persist_dir(
```

Comment:
hmmm, this will likely break backwards-compatibility for sure. 

I think this entire section could be more robust, as they should check for the file before loading it.

Right now I get an error like this when loading a previously saved index: `ValueError: No existing llama_index.graph_stores.simple found at ./nodes_index/graph_store.json, skipping load.`

I can take a stab at this or you can as well :)

Reply:
Dear @logan-markewich, I pushed a commit with all comments covered except for this part, I'll go for lunch first and come back soon for this part.